notebooks/official/feature_store/online_feature_serving_fetching_multiple_entities.ipynb

```diff
@@ -0,0 +1,850 @@
+        "    )\n",
+        ")\n",
+        "df = session.read_gbq_query(QUERY)\n",
+        "df.to_gbq(BQ_TABLE_URI, if_exists=\"replace\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "818a36b9da86"
+      },
+      "source": [
+        "### Create Feature Online Store"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "1f915ddd4669"
+      },
+      "outputs": [],
+      "source": [
+        "fos: FeatureOnlineStore = FeatureOnlineStore.create_bigtable_store(FOS_ID)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "1ee60189d98a"
+      },
+      "source": [
+        "### Create Feature View"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "0dba1c02883c"
+      },
+      "outputs": [],
+      "source": [
+        "fv: FeatureView = fos.create_feature_view(\n",
+        "    FV_ID,\n",
+        "    source=fs_utils.FeatureViewBigQuerySource(\n",
+        "        uri=f\"bq://{BQ_TABLE_URI}\",\n",
+        "        entity_id_columns=[\"entity_id\"],\n",
+        "    ),\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "c71964219962"
+      },
+      "source": [
+        "### Sync the feature view"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "8fb9774db748"
+      },
+      "outputs": [],
+      "source": [
+        "fv_sync = fv.sync()"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "9ee5be32ada4"
+      },
+      "outputs": [],
+      "source": [
+        "# Wait for sync to complete\n",
+        "import time\n",
+        "\n",
+        "while True:\n",
+        "    fv_sync = FeatureView.FeatureViewSync(fv_sync.resource_name)\n",
+        "    if \"final_status\" in fv_sync._gca_resource:\n",
+        "        print(\"FeatureViewSync finished!\")\n",
+        "        break\n",
+        "\n",
+        "    print(\"Waiting for FeatureViewSync...\")\n",
+        "    time.sleep(10)\n",
+        "\n",
+        "# Wait for caches/etc. to expire.\n",
+        "time.sleep(60)\n",
+        "\n",
+        "FeatureView.FeatureViewSync(fv_sync.resource_name)._gca_resource"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "f9ecae3005df"
+      },
+      "source": [
+        "### Reading multiple entities\n",
+        "\n",
+        "In the following sections we'll explore a few different ways to read multiple entities. Using the FetchFeatureValues API results in higher latency due to sequential calls - using threads slightly reduces latency. Using the StreamingFetchFeatureValues API reduces latency greatly as batching of reads is done by the Feature Store server."
```

Comment:
Avoid the use of the first person. I'd go with something like "The following sections demonstrate different ways to read multiple entities. ..."